sbin/Release.sh

```diff
@@ -14,44 +14,48 @@
 #
 
 npm install
-# Expect a naming convention for build artifacts to follow this pattern:
-# OpenJDK<version>_<arch>_<os>_<timestampOrTag>.<extension>
-# Examples: OpenJDK8_x64_Windows_201813060547.zip, 
-#           OpenJDK8_x64_LinuxLH_201813060547.tar.gz, 
-#           OpenJDK10_aarch64_Linux_201813060547.tar.gz.sha256.txt,
-#           OpenJDKamber_x64_Linux_201813061304.tar.gz	
+
```

Comment:
Can you put an example back in to show what we're translating i.e. from something YYYYY-DD-MM to YYYY-MM-DD - not obvious to the casual user what's going on here (and you've removed the examples that were there previously)